app_service_linux_vnet_integration.md

```diff
@@ -6,5 +6,7 @@ During Preview you will need to modify your application in order to integrate wi
 ~~~
 app.listen(process.env.PORT); 
 ~~~
+[ASPNETCORE_URLS](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/web-host?view=aspnetcore-3.1#server-urls) is the recommended way to configure ASP.NET Core docker image and here is the [DockerFile](https://github.com/dotnet/dotnet-docker/blob/7c18c117d9bd2d35e30cdb4a1548ac14b9f0f037/3.0/aspnetcore-runtime/nanoserver-1809/amd64/Dockerfile) example.
```

Comment:
The URL for ASPNETCORE_URLS shoudn't contain the `/en-us/` segment, please change this to:

- https://docs.microsoft.com/aspnet/core/fundamentals/host/web-host?view=aspnetcore-3.1#server-urls

Reply:
The change is completed, thanks